index.js

```diff
@@ -1,46 +1,78 @@
 /***** Beginning of Starter Code ****/
 
-const playerContainer = document.querySelector(".player-container")
+const playerContainer = document.querySelector(".player-container");
 
 // render one player to the DOM
 function renderPlayer(player) {
   // create an element for the outer div
-  const playerDiv = document.createElement("div")
+  const playerDiv = document.createElement("div");
 
   // set attributes on the outer div
-  playerDiv.className = "player"
-  playerDiv.dataset.number = player.number
+  playerDiv.className = "player";
+  playerDiv.dataset.number = player.number;
 
   // use innerHTML to create any child elements of the div
   playerDiv.innerHTML = `
     <h3>${player.name} (<em>${player.nickname}</em>)</h3>
     <img src="${player.photo}" alt="${player.name}">
     <p class="likes">${player.likes} likes</p>
     <button class="like-button">❤️</button>
-  `
+  `;
 
   // append the element to the container
-  playerContainer.append(playerDiv)
+  playerContainer.append(playerDiv);
+  
+  // deliverable 3
+  const likeButton = playerDiv.querySelector("button.like-button")
+  likeButton.addEventListener("click", function() {
+    const likePhrase = playerDiv.querySelector("p.likes").textContent
+    let likeCount = parseInt(likePhrase.replace(" likes",""));
+    likeCount++;
+    playerDiv.querySelector("p.likes").textContent = `${likeCount} likes`
```

Comment:
Since your function here has access to the `player` variable from the outer scope, you can actually just do this:

```js
    player.likes++;
    playerDiv.querySelector("p.likes").textContent = `${player.likes} likes`
```